hx_lti_initializer/templates/tx/detail.html

```diff
@@ -60,7 +61,14 @@
                 <a href="{% url 'hx_lti_initializer:course_admin_hub' %}">Home</a>
         </div>
   		<div class="col-xs-7" id="leftCol">
-			<h1>{{ target_object.target_title | safe }}</h1>
+        <div class="instructions-container" style="display:{% if instructions %}block{% else %}none{% endif %};">
+    			<div class="instructions-title">
+    				Instructions
+    				<span href="#" class="toggle-instructions" data-toggle="collapse" data-target=".instructions-body">Collapse Instructions</span>
+    			</div>
+    			<div class="instructions-body collapse in" aria-expanded="true">{{instructions | safe}}</div>
```

Comment:
The ```instructions``` template variable probably shouldn't be marked as [safe](https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#safe). If the user adds broken HTML to the instructions field, that will in turn break the rest of the page, since it won't be escaped. This would be better:

```
{{instructions | escape | linebreaks}}
```